ember-cli-build.js

```diff
@@ -6,428 +6,399 @@
 //
 // DISABLE_ES3=true DISABLE_JSCS=true DISABLE_JSHINT=true DISABLE_MIN=true DISABLE_DEREQUIRE=true ember serve --environment=production
 
-var fs   = require('fs');
-var path = require('path');
-
-var EmberBuild  = require('emberjs-build');
-var getPackages = require('./lib/packages');
-var getGitInfo  = require('git-repo-info');
-
-var applyFeatureFlags = require('babel-plugin-feature-flags');
-var filterImports     = require('babel-plugin-filter-imports');
-
-var vendoredPackage    = require('emberjs-build/lib/vendored-package');
-var htmlbarsPackage    = require('emberjs-build/lib/htmlbars-package');
-var replaceVersion     = require('emberjs-build/lib/utils/replace-version');
-
-var Funnel = require('broccoli-funnel');
-var Rollup = require('broccoli-rollup');
-var mergeTrees = require('broccoli-merge-trees');
-var replace = require('broccoli-string-replace');
-var uglify = require('broccoli-uglify-sourcemap');
-
-var rollupEnifed = {
-  transformBundle(code, options) {
-    return {
-      code: code.replace(/\bdefine\(/, 'enifed('),
-      map: { mappings: null }
-    };
-  }
-};
+const MergeTrees = require('broccoli-merge-trees');
+const babelHelpers = require('./broccoli/babel-helpers');
+const bootstrapModule = require('./broccoli/bootstrap-modules');
+const addon = require('./broccoli/addon');
+const concat = require('./broccoli/concat-bundle');
+const testIndexHTML = require('./broccoli/test-index-html');
+const toAMD = require('./broccoli/to-amd');
+const toES5 = require('./broccoli/to-es5');
+const minify = require('./broccoli/minify');
+const lint = require('./broccoli/lint');
+const {
+  routerES,
+  jquery,
+  internalLoader,
+  qunit,
+  glimmerDependencyInjectionES,
+  emberGlimmerES,
+  handlebarsES,
+  rsvpES,
+  simpleHTMLTokenizerES,
+  backburnerES,
+  dagES,
+  routeRecognizerES,
+  emberPkgES,
+  glimmerPkgES,
+  emberTestsES,
+  nodeModuleUtils,
+  emberVersionES,
+  emberLicense,
+  emberFeaturesES,
+  packageManagerJSONs,
+  nodeTests
+} = require('./broccoli/packages');
```

Comment:
Destructuring requires node 6 😢. I'm ok with requiring node@6 here, but when we ultimately have ember-cli apps building ember-source themselves (when not running a tagged version) we will need to downgrade to support node@4.